monai/utils/to_onehot.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn.functional as f
+
+
+def to_onehot(data, num_classes: int):
```

Comment:
Is this not at least partially a duplicate of monai.networks.utils.one_hot? There isn't a need to restrict the first channel to a size of 1, it's conceivable to define a labelling associated with multichannel data with differing labels for each channel.

Reply:
i think in that multichannel label case, each channel might use a different `num_classes`, we'll need an unstack first and then convert each channel individually